lava-v2-jobs-from-api.py

```diff
@@ -197,20 +195,19 @@ def jinja_render(job):
 
 if __name__ == '__main__':
     parser = argparse.ArgumentParser()
-    parser.add_argument("--token", help="KernelCI API Token")
+    parser.add_argument("--token", help="KernelCI API Token", required=True)
     parser.add_argument("--api", help="KernelCI API URL", default="https://api.kernelci.org")
     parser.add_argument("--storage", help="KernelCI storage URL", default="https://storage.kernelci.org")
-    parser.add_argument("--jobs", help="absolute path to top jobs folder")
-    parser.add_argument("--tree", help="KernelCI build kernel tree")
-    parser.add_argument("--branch", help="KernelCI build kernel branch")
-    parser.add_argument("--describe", help="KernelCI build kernel git describe")
-    parser.add_argument("--config", help="configuration for the LAVA server")
-    parser.add_argument("--section", default="default", help="section in the LAVA config file")
+    parser.add_argument("--lab", help="KernelCI Lab Name", required=True)
+    parser.add_argument("--tree", help="KernelCI build kernel tree", required=True)
+    parser.add_argument("--branch", help="KernelCI build kernel branch", required=True)
+    parser.add_argument("--describe", help="KernelCI build kernel git describe", required=True)
     parser.add_argument("--plans", nargs='+', required=True, help="test plan to create jobs for")
```

Comment:
Same comment about required command line options, they should not be required if they can be read from the config file.